### ISSUE TITLE
Rule `no-mixed-spaces-and-tabs` should not apply inside template literals - followup

### ISSUE THREAD

**Issue by sindresorhus** [eslint/eslint#4190]
https://github.com/eslint/eslint/issues/4054 didn't fix all cases.

-

foo.js:

```js
const foo = `${console}
	  foo`;
```

*The second line is `\t  foo`.*

```
~/dev
❯ eslint --version                                                               
1.7.1
                                                                                                                                               
~/dev
❯ eslint foo.js --env=es6 --rule='no-mixed-spaces-and-tabs: 2'

/Users/sindresorhus/dev/foo.js
  2:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs

✖ 1 problem (1 error, 0 warnings)
```

-

Can you just make the rule ignore *everything* inside a template literal?
